expression/function_traits.go

```diff
@@ -40,7 +40,7 @@ var unFoldableFunctions = map[string]struct{}{
 	ast.RowFunc:   {},
 	ast.Values:    {},
 	ast.SetVar:    {},
-	ast.GetVar:    {},
+	//ast.GetVar:    {},
```

Comment:
It's better directly to remove the line.

Reply:
I will fix this.